<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="4" total="7" passed="1" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-01-05T16:39:44 EST" name="Suite" finished-at="2025-01-05T16:39:51 EST" duration-ms="6290">
    <groups>
    </groups>
    <test started-at="2025-01-05T16:39:44 EST" name="Test" finished-at="2025-01-05T16:39:51 EST" duration-ms="6290">
      <class name="com.luma.loginpagetests.CreateNewCxAccTest">
        <test-method is-config="true" signature="launchSite()[pri:0, instance:com.luma.loginpagetests.CreateNewCxAccTest@6e20b53a]" started-at="2025-01-05T16:39:35 EST" name="launchSite" finished-at="2025-01-05T16:39:44 EST" duration-ms="8953" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchSite -->
        <test-method signature="VerifyCreateNewCxAccTest()[pri:0, instance:com.luma.loginpagetests.CreateNewCxAccTest@6e20b53a]" started-at="2025-01-05T16:39:44 EST" name="VerifyCreateNewCxAccTest" finished-at="2025-01-05T16:39:51 EST" duration-ms="6167" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyCreateNewCxAccTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.luma.loginpagetests.CreateNewCxAccTest@6e20b53a]" started-at="2025-01-05T16:39:51 EST" name="tearDown" finished-at="2025-01-05T16:39:52 EST" duration-ms="942" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.luma.loginpagetests.CreateNewCxAccTest -->
      <class name="com.luma.loginpagetests.LogOutTest">
        <test-method signature="verifyLogOutTest()[pri:1, instance:com.luma.loginpagetests.LogOutTest@5af3afd9]" started-at="2025-01-05T16:39:51 EST" name="verifyLogOutTest" finished-at="2025-01-05T16:39:51 EST" duration-ms="9" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "org.openqa.selenium.WebDriver.findElement(org.openqa.selenium.By)" because "this.driver" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebDriver.findElement(org.openqa.selenium.By)" because "this.driver" is null
at com.luma.pages.CommonPage.clickUserHeader(CommonPage.java:30)
at com.luma.loginpagetests.LogOutTest.verifyLogOutTest(LogOutTest.java:14)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLogOutTest -->
      </class> <!-- com.luma.loginpagetests.LogOutTest -->
      <class name="com.luma.loginpagetests.LoginTest">
        <test-method signature="verifyLoginTest()[pri:3, instance:com.luma.loginpagetests.LoginTest@661972b0]" started-at="2025-01-05T16:39:51 EST" name="verifyLoginTest" finished-at="2025-01-05T16:39:51 EST" duration-ms="0" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "org.openqa.selenium.WebDriver.findElement(org.openqa.selenium.By)" because "this.driver" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebDriver.findElement(org.openqa.selenium.By)" because "this.driver" is null
at com.luma.pages.HomePage.clickSignInLink(HomePage.java:25)
at com.luma.loginpagetests.LoginTest.verifyLoginTest(LoginTest.java:15)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginTest -->
      </class> <!-- com.luma.loginpagetests.LoginTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
